packages/block-library/src/navigation/edit/menu-inspector-controls.js

```diff
@@ -49,12 +51,20 @@ const ExperimentMainContent = ( {
 		return <Spinner />;
 	}
 
+	const label = navigationMenu
+		? sprintf(
+				/* translators: %s: The name of a menu. */
+				__( 'Navigation menu: %s' ),
```

Comment:
Okay, I think this is close. Generally, I would never advocate for verbosity, but I think a longer label could actually be better in this specific example.

```suggestion
				__( 'Structure for navigation menu: %s' ),
```